recipes/vulkan-memory-allocator/all/conanfile.py

```diff
@@ -10,20 +11,42 @@ class VulkanMemoryAllocatorConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     description = "Easy to integrate Vulkan memory allocation library."
     topics = ("vulkan", "memory-allocator", "graphics")
-    requires = ("vulkan-headers/1.2.182")
-    no_copy_source = True
+    settings = "os", "arch", "compiler", "build_type"
 
     @property
     def _source_subfolder(self):
         return "source_subfolder"
 
+    @property
+    def _min_cppstd(self):
+        return "11" if tools.Version(self.version) < "3.0.0" else "14"
+
+    def export_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def requirements(self):
+        self.requires("vulkan-headers/1.3.211.0")
```

Comment:
```suggestion
```

Reply:
Header vulkan memory allocator includes vulkan header, so what you propose is a violation of any package manager paradigm.

Reply:
I see, thanks for pointing that out.

Do you have any suggestions of a correct solution to the problem that I described?

Is adding an option to conditionally override the requirement of Vulkan headers also a violation?

Reply:
Sorry, but you are mixing unmanaged dependencies with conan managed dependencies, you should ask on Slack or https://github.com/conan-io/conan, but it's out of scope of this PR.

Reply:
OK, thanks!